Source/WebInspectorUI/UserInterface/Views/SingleSidebar.css

```diff
@@ -47,6 +47,16 @@
     border-inline-end: 1px solid var(--border-color);
 }
 
-.single-sidebar.trailing {
+body:not(.narrow) .single-sidebar.trailing {
     border-inline-start: 1px solid var(--border-color);
 }
+
+body.narrow .sidebar.trailing .single-sidebar {
```

Comment:
nit: technically speaking this doesn't know about whether it's a child of `.sidebar.trailing`, so i think this should just be `body.narrow .single-sidebar.trailing`